recipes/cif2cell/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 2268a2d6d26650433a1c5e51c98e400cf92e458036879e231088555d75e3e8bf
 
 build:
-  noarch: python
+  skip: True  # [unix]
```

Comment:
this only runs on windows?

Reply:
No @CJ-Wright, but since only the Windows build fails, I didn't want to waste the unix resources in the meantime.

Reply:
No @CJ-Wright , I just didn't want to waste the the resources until the Windows build is fixed.